README.md

```diff
@@ -119,6 +119,8 @@ The Liberty Maven Plugin provides the following goals.
 | [undeploy](docs/undeploy.md#undeploy) | Undeploy an application to a Liberty server. The server instance must exist and must be running. |
 | [uninstall-feature](docs/uninstall-feature.md#uninstall-feature) | Uninstall a feature from the Liberty runtime. |
 
+*The `dev`, `devc`, and `generate-features` goals have a runtime dependency on IBM WebSphere Application Server Migration Toolkit for Application Binaries, which is separately licensed under ILAN. Link to license: https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/license/wamt/en.html   
```

Comment:
is this URL really where we want to send people for MT? It's a static copy of the license. Seems like a strange jump off point to obtain MT if the user doesn't already have it. Yes they need to be aware of license terms but this information is more aligned with legal terms than user experience. I realize we might be obligated to share the license info in this format, in which case disregard this comment. But if not, I wonder if there is a more user-friendly resource we could include here.